pkg/kademlia/kademlia.go

```diff
@@ -210,9 +210,11 @@ func (k *Kad) manage() {
 // binSaturated indicates whether a certain bin is saturated or not.
 // when a bin is not saturated it means we would like to proactively
 // initiate connections to other peers in the bin.
-func binSaturated(bin, depth uint8, peers *pslice.PSlice) bool {
+func binSaturated(bin, depth uint8, peers, connected *pslice.PSlice) bool {
```

Comment:
Looks like that depth argument is no longer used.

Reply:
true